botocove/cove_host_account.py

```diff
@@ -26,10 +29,26 @@ def __init__(
         policy_arns: Optional[List[str]],
         assuming_session: Optional[Session],
         org_master: bool,
+        thread_workers: int,
+        regions: Optional[List[str]],
     ) -> None:
 
+        self.thread_workers = thread_workers
+
         self.sts_client = self._get_boto3_sts_client(assuming_session)
         self.org_client = self._get_boto3_org_client(assuming_session)
+        self.target_regions: Sequence[Optional[str]] = [None]
+
+        if regions is not None:
+            if (
+                "ALL" in regions
```

Comment:
I'd say for the first version of the feature, don't bother "ALL" as a shorthand.

I would probably never use this because its meaning is specific to this library and I as a user probably have a different idea of what all means for my use case.

Example 1: In my organization operates in 5 regions and blocks the rest via SCP. So I will at most pass the list of 5 regions to the core function.

Example 2: I need to query Workmail administrators. The service is available only in eu-west-1, us-east-1, and us-west-1. So I will pass a list of those regions to the function. I don't need botocove to know which services are available in which regions.

Reply:
Thanks, good feedback - anyone willing to use a library to write boto3 functions is likely going to be willing to fetch their own region list should they desire one :)